x-pack/plugins/case/common/api/cases/commentable_case.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+import { CaseAttributesRt } from './case';
+import { CommentResponseRt } from './comment';
+import { SubCaseAttributesRt, SubCaseResponseRt } from './sub_case';
+
+export const CollectionSubCaseAttributesRt = rt.intersection([
+  rt.partial({ subCase: SubCaseAttributesRt }),
+  rt.type({
+    case: CaseAttributesRt,
+  }),
+]);
+
+export const CollectWithSubCaseResponseRt = rt.intersection([
+  CaseAttributesRt,
+  rt.type({
+    id: rt.string,
+    totalComment: rt.number,
+    version: rt.string,
+  }),
+  rt.partial({
+    subCase: SubCaseResponseRt,
```

Comment:
Again I am missing something 😄 . Why the collection have only one subcase and not an array of subcases?